packages/prometheus-clamav/PKGBUILD

```diff
@@ -0,0 +1,103 @@
+# _gitname="clamav-prometheus-exporter-master"
+pkgname="carbonio-prometheus-clamav-exporter"
+pkgdesc="Prometheus-Exporter for ClamAV"
+pkgver="0.1.0"
+maintainer="Zextras <packages@zextras.com>"
+pkgrel="1"
+section="mail"
+priority="optional"
+arch=('x86_64')
+url="https://zextras.com"
+license=("Apache-2.0")
+source=(
+        "https://github.com/r3kzi/clamav-prometheus-exporter/archive/refs/heads/master.zip"
+        "${pkgname}.service"
+        "${pkgname}-consul.hcl"
+        "prometheus.sysusers"
+        "${pkgname}.default"
+)
+
+depends=(
+  "systemd"
+)
+
+conflicts=(
+  'prometheus-clamav-exporter'
+)
+
+makedepends=(
+  "git"
+)
+
+sha256sums=(
+  "SKIP"
+  "SKIP"
+  "SKIP"
+  "SKIP"
+  "SKIP"
+)
+
+ build(){
+  cd "${srcdir}/clamav-prometheus-exporter-master/"
+
+    export CGO_CPPFLAGS="${CPPFLAGS}"
+    export CGO_CFLAGS="${CFLAGS}"
+    export CGO_CXXFLAGS="${CXXFLAGS}"
+    export CGO_LDFLAGS="${LDFLAGS}"
+    export GOFLAGS="-buildmode=pie -trimpath -ldflags=-linkmode=external -mod=readonly"
+
+    go build -o "${srcdir}/clamav-prometheus-exporter-master/carbonio-prometheus-clamav-exporter"
```

Comment:
if you already are in `"${srcdir}/clamav-prometheus-exporter-master` path (line 41), doing 
`go build -o carbonio-prometheus-clamav-exporter` should be enough (line 49)